modal-go/config.go

```diff
@@ -120,3 +106,22 @@ func environmentName(environment string) string {
 func imageBuilderVersion(version string) string {
 	return firstNonEmpty(version, defaultProfile.ImageBuilderVersion, "2024.10")
 }
+
```

Comment:
Should `environmentName` and `imageBuilderVersion` a few lines above be using `clientProfile`?